firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/ApkInstaller.java

```diff
@@ -93,7 +97,7 @@ void handleAppResume(Activity activity) {
         cachedInstallApkPath = path;
       } else {
         // only start the install activity if current Activity is in the foreground
-        startInstallActivity(path, currentActivity);
+        startInstallActivity(path);
```

Comment:
Yes it can!

Reply:
Can we now start the Install Activity regardless of whether or not the customer's app is in the foreground?

Reply:
Good question! Currently working on trying to see if there is a set up to make that work. In its current set up if the user backgrounds the app it seems to hang.

Reply:
It seems like this is not possible due to Install Activity relying on an activity to show unknown sources UI.

Reply:
That is surprising. I'd think that once the InstallActivity starts, then it's in the foreground and we can show the dialog. That's what happened when you were calling UpdateApp from a background thread, which worked.

What error/stacktrace are you seeing in the logs?

Reply:
I get a windowLeaked error. I can repro and show the error message. I believe that was because the app itself wasn't in the background, but once you background the app it wants to display the unknown sources UI but since the app is in the background you get this leaked error.

Reply:
Why does the app get backgrounded? Maybe we could look at it together over VC.

Reply:
We discussed this in person. Previously, backgrounding the app after starting the basic config would fail the task correctly. With this change it instead hangs the task.

I'm fine punting on this for now.

Reply:
To clarify it hangs if we don't depend on the currentActivity at all with checking what to do with InstallActivity.